### PR TITLE
Started working on the admin dashboard. Added admin controller and model

### PR DESCRIPTION
**Pull request by ProsperChiduku**

### CODE REVIEW

File: app/views/category_admin/_table.html.erb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  <table class="table">
+  <thead>
+    <tr>
+      <th><abbr title="ID">ID</abbr></th>
+      <th><abbr title="Category Name">Category Name</abbr></th>
+      <th><abbr title="Added By">Added By</abbr></th>
+      <th>Options</th>
+      <th>
+        <div id="add-button">
+          <button class="button is-success"><%= link_to "Add Category", "new", id: 'link-color' %></button>
```
Comment (aaronmyatt) [inline app/views/category_admin/_table.html.erb:13]: need to fix the url here `new` should point to `category_admin/new`

File: app/views/drilldown_admin/_table.html.erb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    <table class="table">
+  <thead>
+    <tr>
+      <th><abbr title="ID">ID</abbr></th>
+      <th><abbr title="Category Name">Category Name</abbr></th>
+      <th><abbr title="Added By">Drilldown Name</abbr></th>
+      <th>Options</th>
+      <th>
+        <div id="add-button">
+          <button class="button is-success"><%= link_to "Add Category", "new", id: 'link-color' %></button>
```
Comment (aaronmyatt) [inline app/views/drilldown_admin/_table.html.erb:13]: fix the path here too, `new` fails

**Review by aaronmyatt — CHANGES_REQUESTED** [review]

**Comment by aaronmyatt** [comment]
@ProsperMindvalley @pavi304 we definitely want to fix the layout stuff too. It's getting quite messy seeing all that repeated code.

File: app/controllers/category_admin_controller.rb [hunk 3]
```diff
@@ -0,0 +1,48 @@
+class CategoryAdminController < ApplicationController
+  def index
+      @category_admins = CategoryAdmin.all
+  end
+  def show
```
Comment (prosperityk) [inline app/controllers/category_admin_controller.rb:5]: Remove me

File: app/controllers/drilldown_controller.rb [hunk 4]
```diff
@@ -2,21 +2,28 @@ class DrilldownController < ApplicationController
...
-      end
 
-       @drilldown1 = drilldown[0..3]
-       @drilldown2 =drilldown[4..7]
-       @drilldown3 =drilldown[8..11]
-       @drilldown4 =drilldown[12..15]
+      drilldowns = DrilldownAdmin.take(16)
+      split = drilldowns.each_slice(4).to_a
+
+      # session[:category].each do |category|
```
Comment (prosperityk) [inline app/controllers/drilldown_controller.rb]: Remove all these comments

File: app/controllers/questions_controller.rb [hunk 5]
```diff
@@ -1,20 +1,27 @@
...
-      end
-    end
-    @questions1 = question[0..1]
-    @questions2 = question[2..3]
-    @questions3 = question[4..5]
-    @questions4 = question[6..7]
+
+    all_questions = QuestionAdmin.take(8)
+    split = all_questions.each_slice(2).to_a
+    # question=[]
```
Comment (prosperityk) [inline app/controllers/questions_controller.rb]: Please Remove Me

File: app/views/layouts/application.html.erb [hunk 6]
```diff
@@ -7,6 +7,7 @@
 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.7.2/css/bulma.min.css">
```
Comment (prosperityk) [inline app/views/layouts/application.html.erb:10]: Use Gem for this

File: app/controllers/drilldown_controller.rb (hunk 4 above)
Comment (pavikunusilin) [reply app/controllers/drilldown_controller.rb]: done.

File: app/controllers/questions_controller.rb (hunk 5 above)
Comment (pavikunusilin) [reply app/controllers/questions_controller.rb]: done.

**Pull request merged**